fe/fe-core/src/main/java/com/starrocks/scheduler/MvTaskRunProcessor.java

```diff
@@ -359,17 +372,17 @@ private void refreshMv(TaskRunContext context, MaterializedView materializedView
                     new PartitionNames(false, tablePartitionNames.stream().collect(Collectors.toList())));
             // e.g. insert into mv partition(p1,p2) select * from table partition(p3)
             StringBuilder insertSqlBuilder = new StringBuilder();
-            insertSqlBuilder.append("insert into ");
+            insertSqlBuilder.append("insert overwrite ");
             insertSqlBuilder.append(materializedView.getName());
             insertSqlBuilder.append(" partition(" + mvPartitionName + ") ");
             insertSqlBuilder.append(AST2SQL.toString(queryStatement));
             String insertIntoSql = insertSqlBuilder.toString();
-            execInsertStmt(insertIntoSql, context);
+            execInsertStmt(insertIntoSql, context, materializedView);
```

Comment:
better to Generate ExecPlan here manually and we can get the source partitions' version to update the base table partition version map

Reply:
I think this PR mainly uses insert overwrite, by the way to fix other problems, can we use a new PR for this change?